deployment/ansible/roles/cac-tripplanner.app/tasks/main.yml

```diff
@@ -0,0 +1,28 @@
+---
+- name: Install packages
+  apt: name={{ item }} state=present
+  with_items:
+    - postgresql-9.3
+    - postgresql-9.3-postgis-2.1
+    - libpq-dev
+    - python-dev
+    - python-psycopg2
+
+- name: Install pip packages
+  pip: name={{ item }}
+       virtualenv=/app_env
+  with_items:
```

Comment:
+1 to requiring versions; we can do something like:
`- { name: Django, version: 1.7.1 }` and then reference them with `{{ item.version }}`